Improve Cloud Starter - New Relic config instructions (#6809)
magento/devdocs: src/cloud/project/new-relic.md

  - Services
---

[New Relic for application performance management (APM)] is a software analytics product that helps you analyze and improve application interactions. New Relic APM is available to all {{site.data.var.ece}} accounts and provides the following features:

-  **Focus on specific transactions**—Actively mark and monitor key customer actions in your site, such as adding to the cart, checking out, or processing a payment.
-  **Database query monitoring**—Locate and monitor database queries affecting performance.
-  **App Map**—View all application dependencies within your Magento site, extensions, and external services.

> -  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event. [Apdex score].

Reviewer: ```suggestion
-  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event. See [Apdex score].
```

-  For Starter accounts, you have a New Relic license key that supports up to three environments. You must add the key to your environment configurations manually. The license key is not pre-provisioned on Starter environments.

### Configure New Relic for Starter environments

For Starter environments, you must add the New Relic license key to the environment configuration to enable New Relic reporting. We recommend adding the key to the Staging and Production (master) environments and one other environment of your choice. Only the New Relic license key is required for configuration. You can find information about additional configuration options in the [New Relic reporting] topic in the _Magento User Guide_.

{:.bs-callout-warning}
Updating the environment configuration triggers a redeployment, which takes your site offline until deployment completes. For Production environments, we recommend completing this work during off-peak hours to avoid service disruptions. See [Working with variables].

> {:.prerequisites}

Author: ```suggestion
{:.procedure}
Prerequisites
```

{:.bs-callout-warning}
Updating the environment configuration triggers a redeployment, which takes your site offline until deployment completes. For Production environments, we recommend completing this work during off-peak hours to avoid service disruptions. See [Working with variables].

{:.procedure}
Prerequisites

-  Log in credentials for the Magento Cloud project account page, or for the New Relic account associated with your project
-  [Admin level access]({{site.baseurl}}/cloud/project/user-admin.html) to the Starter environments to configure, or credentials to access the [Magento Admin](https://docs.magento.com/m2/ce/user_guide/system/permissions.html) for the environment.

{:.procedure}
To configure New Relic on a Starter environment:

1. Find your New Relic license key from your Magento account page, or [from your New Relic account page]:

   -  Open your [Magento account page].

   -  On the _Projects_ tab, find your project.

   -  Click **View Details** to see the project infrastructure information.

   -  Expand the **New Relic Service** section to view the license key.

   -  Copy the license key.

> 1. Add the New Relic license key to an environment using an environment variable, or [add it from the Magento Admin]:

Reviewer: ```suggestion
1. Add the New Relic license key to an environment using the magento-cloud CLI, or [add it from the Magento Admin]:
```

Need to be specific. Saying environment variable may make some people think about the .magento.env.yaml file. or the Project Web UI variables list.

Do not modify the Adobe-generated alert policies. We will automatically regenerate and improve them over time, which will overwrite any customizations you make.

If you want to modify the thresholds of an alert condition, create a new alert policy. Then, copy the alert condition to the new policy. See [Update policies or conditions] in the New Relic documentation.

{:.bs-callout-tip}
See [Alerts concepts and workflow] in the New Relic documentation for more detailed information about Alerts, alert policies, and notification channels.

> 

Reviewer: Might be nice to have a comment here.
```suggestion

<!-- link definitions -->

```
